translations/monero-core_fr.ts

```diff
@@ -1747,7 +1753,9 @@ Veuillez mettre à jour ou vous connecter à un autre démon</translation>
         <location filename="../pages/TxKey.qml" line="97"/>
         <source>Generate a proof of your incoming/outgoing payment by supplying the transaction ID, the recipient address and an optional message. 
 For the case of outgoing payments, you can get a &apos;Spend Proof&apos; that proves the authorship of a transaction. In this case, you don&apos;t need to specify the recipient address.</source>
-        <translation type="unfinished"></translation>
+        <translation> Générer une preuve de votre paiement en fournissant le ID de transaction, l'addresse destinaire et un message facultatif.
+Pour le cas des paiements sortants, vous ouvez obtenir un &apos;Preuve dedépense&apos; qui prouve l'auteur d'une transaction. En ce cas, il ne faut pas spécifier une addresse destinaire.
```

Comment:
Pour le cas des paiements sortants, vous pouvez obtenir un &apos;Preuve de Dépense&apos; qui prouve l'auteur d'une transaction. Dans ce cas, il n'est pas nécessaire de spécifier l'adresse du destinataire.

Reply:
"Paiements sortants" et "prouve l'auteur" don't sound right. How about:  
> Pour les paiements émis, vous pouvez obtenir une "preuve de dépense" qui prouve l'identité de l'auteur de la transaction. [etc.]

Reply:
Yes I agree with both of your remarks.
I would prefer "Preuve de Paiement" comparer to "Preuve de dépense" but I am not sure if it would change the actual meaning. (why isn't it "payment proof" in this case ?)

Reply:
Yes "preuve de paiement" feels more natural but I think there's a technical difference between transaction proof and spend proof: imagine you receive 11 XMR from someone and then pay someone else 10 XMR. The wallet will send 11 XMR and return 1 XMR change. The payment or transaction is the 10 XMR but the spend proof will show the 11 XMR only I believe. I haven't use the GUI so not sure if that's the case though.